sentry-core/src/main/java/io/sentry/core/transport/HttpTransport.java

```diff
@@ -30,14 +31,15 @@
 import java.util.Date;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
+import java.util.zip.GZIPOutputStream;
 import javax.net.ssl.HttpsURLConnection;
 import org.jetbrains.annotations.ApiStatus;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 /**
  * An implementation of the {@link ITransport} interface that sends the events to the Sentry server
- * over HTTP(S) in UTF-8 encoding.
+ * over HTTP(S) in gzip encoding.
```

Comment:
gzip and UTF-8 are orthogonal concepts.
The SDK is sending a UTF-8-encoded JSON representation of an event compressed with gzip 😄 

```suggestion
 * over HTTP(S).
```

Is it relevant to talk about UTF-8/JSON/gzip in the javadoc?! I suspect not.

Reply:
that's true, yeah let's clean this up.